Hardware/CPU/IntelCPU.cs

```diff
@@ -105,7 +105,7 @@ public IntelCPU(int processorIndex, CPUID[][] cpuid, ISettings settings)
                       default:
                         tjMax = Floats(85 + 10); break;
                     }
-                    tjMax = Floats(80 + 10); break;

```

Comment:
I don't think this should be in this commit?

Reply:
i revert this with the next commit